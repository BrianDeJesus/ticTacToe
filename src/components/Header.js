import React from 'react';

const Header = () => (
    <div className="App-header">
        <a href="https://github.com/BrianDeJesus/ticTacToe" className="Github-link">View on Github</a>
        <h1 className="App-title">Tic-Tac-Toe</h1>
    </div>
);

export default Header;