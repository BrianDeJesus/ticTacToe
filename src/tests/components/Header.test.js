import React from 'react';
import Header from '../../components/Header';
import renderer from 'react-test-renderer';

test('should render Header', () => {
    const component = renderer.create(<Header />).toJSON();
    expect(component).toMatchSnapshot();
});