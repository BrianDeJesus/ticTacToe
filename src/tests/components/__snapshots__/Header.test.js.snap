// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header 1`] = `
<div
  className="App-header"
>
  <a
    className="Github-link"
    href="https://github.com/BrianDeJesus/ticTacToe"
  >
    View on Github
  </a>
  <h1
    className="App-title"
  >
    Tic-Tac-Toe
  </h1>
</div>
`;
